ingestors/kafka/src/main/scala/hydra/kafka/programs/CreateTopic.scala

```diff
@@ -75,13 +75,22 @@ final class CreateTopicProgram[F[_]: Bracket[*[_], Throwable]: Sleep: Logger](
       subject: Subject,
       topicDetails: TopicDetails
   ): Resource[F, Unit] = {
+    val onFailure: (Throwable, RetryDetails) => F[Unit] = {
+      (error, retryDetails) =>
+        Logger[F].info(
+          s"Retrying due to failure: $error. RetryDetails: $retryDetails"
```

Comment:
Should we add something about this being a retry on Kafka topic creation? I want to make sure we can tell what is being retried here